.github/workflows/default.yml

```diff
@@ -0,0 +1,24 @@
+name: default build
+
+on:
+  push:
+    branches:
+      - master
+    tags:        
+      - '*'
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: gitbucket
+          password: ${{ secrets.DOCKER_PASSWORD }}

```

Comment:
We need to set `DOCKER_PASSWORD` to use these workflow. Got it. 👍

Reply:
Yes! Please!